Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/UpdateAcpiTable.c

```diff
@@ -434,7 +438,24 @@ PlatformUpdateAcpiTable (
   } else if (Table->Signature == EFI_BDAT_TABLE_SIGNATURE) {
     UpdateBdatAcpiTable (Table, LdrGlobal->FspHobList);
     DEBUG ((DEBUG_INFO, "Updated BDAT Table in AcpiTable Entries\n"));
-  } else if (Table->Signature == EFI_ACPI_6_1_LOW_POWER_IDLE_TABLE_STRUCTURE_SIGNATURE){
+  } else if (Table->Signature == EFI_ACPI_6_1_FIXED_ACPI_DESCRIPTION_TABLE_SIGNATURE) {
+    DEBUG ( (DEBUG_INFO, "Updated FADT Table entries in AcpiTable\n") );
+    {
+      EFI_ACPI_6_1_FIXED_ACPI_DESCRIPTION_TABLE *FadtPointer;
+
+      FadtPointer = (EFI_ACPI_6_1_FIXED_ACPI_DESCRIPTION_TABLE *) Table;
+      //
+      // The Flags field within the FADT (offset 112)
+      //   1) will have a new Low Power S0 Idle Capable ACPI flag (bit offset 21).
+      //
+      if (FeaturesCfgData != NULL) {
+        if (FeaturesCfgData->Features.LowPowerIdle != 0 && PchIsSciSupported() != 1){
+          DEBUG ( (DEBUG_INFO, "Enabled Low Power S0 Idle Capable ACPI flag\n") );
+          FadtPointer->Flags = (BIT21 | FadtPointer->Flags);

```

Comment:
Should we use a  macro definition for BIT21 to make it more readable ?

Reply:
Noted and uploaded revised patch